recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
It isn't necessary I suppose. Though I'm not totally against it either. It may make for recipes that are easier to reuse for new packages added later, which means they are more likely to be close to our linting specs.